packages/core/package.json

```diff
@@ -18,7 +18,7 @@
     "generate:contracts": "abi-gen --abis 'src/artifacts/**/*.json' --template 'src/templates/contract/contract.handlebars' --partials 'src/templates/contract/partials/**/*.handlebars' --output 'src/contracts/generated/wrappers/'",
     "generate:artifacts": "node ../dev-utils/build/scripts/generate-from-files/bin.js 'src/artifacts/**/*.json' 'src/templates/artifacts.handlebars' 'src/contracts/generated/artifacts.ts'",
     "generate:events": "node ../dev-utils/build/scripts/generate-from-files/bin.js 'src/artifacts/**/*.json' 'src/templates/events.handlebars' 'src/contracts/generated/events.ts'",
-    "generate:multisig": "abi-gen --abis 'src/artifacts/Newsroom.json' --template 'src/templates/contract/multisigproxy.handlebars' --partials 'src/templates/contract/partials/**/*.handlebars' --output 'src/contracts/generated/multisig/'",
+    "generate:multisig": "abi-gen --abis 'src/artifacts/@(Newsroom|EIP20|CivilTCR).json' --template 'src/templates/contract/multisigproxy.handlebars' --partials 'src/templates/contract/partials/**/*.handlebars' --output 'src/contracts/generated/multisig/'",
```

Comment:
How about doing it this way so it can be reused, easily noticable

```json
{
  "generate:multisig": "... 'src/artifacts/${npm_package_config_multsigGlob}.json' ..."
},
"config: {
  "multisigGlob": "@(Newsroom|EIP20|CivilTCR)"
}

Reply:
that works?

Reply:
@walfly Check out root package.json's ganache script